bundles/org.openhab.binding.siemenshvac/src/main/java/org/openhab/binding/siemenshvac/internal/converter/type/NumericTypeConverter.java

```diff
@@ -52,28 +63,120 @@ protected boolean toBindingValidation(Type type) {
     }
 
     @Override
-    protected boolean fromBindingValidation(JsonElement value, String type) {
+    protected boolean fromBindingValidation(JsonElement value, String unit, String type) {
         return true;
     }
 
     @Override
-    protected DecimalType fromBinding(JsonElement value, String type, ChannelType tp, Locale locale)
+    protected State fromBinding(JsonElement value, String unit, String type, ChannelType tp, Locale locale)
             throws ConverterException {
         if ("----".equals(value.getAsString())) {
             return new DecimalType(0);
         } else {
-            return new DecimalType(value.getAsDouble());
+            double dValue = value.getAsDouble();
+
+            String itemType = tp.getItemType();
+
+            if (itemType != null) {
+
+                if (itemType.equals("Number:Temperature")) {
+                    Unit<Temperature> targetUnit = null;
+
+                    if (unit.equals("°C")) {
+                        targetUnit = SIUnits.CELSIUS;
+                    } else if (unit.equals("°C*min")) {
```

Comment:
Is it really just °C? For °C/min I would expect something like cooling rate (Celsius per minute), but I don't understand why °C would be provided as °C*min?

Reply:
You're right, there is something wrong here.
I will introduce a new Unit/Quantity for this.
I think it will be more a TemperatureChangeRate then a CollingRate.
It's mostly use to said thing like this:

"if temperature change rate is less then x° / min, then trigger the electrical resistance, or boiler to increase the temperature change rate."

Reply:
Resolved by dropping UoM for this value.